steps/codecov/3.3.0/step.yml

```diff
@@ -0,0 +1,69 @@
+title: Codecov
+summary: Upload your code coverage files to Codecov.io
+description: |-
+  Reduce technical debt with visualized test performance,
+  faster code reviews and workflow enhancements.
+  Now you can upload your code coverage files to Codecov every time you kick off a build!
+website: https://codecov.io
+source_code_url: https://github.com/codecov/codecov-bitrise
+support_url: https://community.codecov.io
+published_at: 2023-06-27T09:35:26.180619-04:00
+source:
+  git: https://github.com/codecov/codecov-bitrise.git
+  commit: 133598e022aca08c75e8af1d32a4a794010336c0
+type_tags:
+- test
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+inputs:
+- CODECOV_TOKEN: null
+  opts:
+    description: |
+      Codecov.io repository upload token
+    is_required: true
+    is_sensitive: true
+    title: Codecov.io token
+- OS: null
+  opts:
+    description: |
+      Name of the OS being used (options are "alpine" | "linux" | "macos")
+    is_required: true
+    is_sensitive: false
+    title: OS Name
+- VERSION: latest
+  opts:
+    description: |
+      Version of the Codecov Uploader to use (e.g. `v0.6.1`)
+    is_required: false
+    is_sensitive: false
+    title: Uploader Version
+- XCODE_ARCHIVE_PATH: null
```

Comment:
@thomasrockhu-codecov out of curiosity, why is this input deprecated?

Reply:
@godrei this is an older way we handled xcode coverage conversion. It is slower and doesn't always behave the way our users expect